### PR TITLE
Translate Ruby OpenSSL Hostname Verification post to Chinese for zh_cn site

### PR DESCRIPTION
**Pull request by springwq**
Related #1074

### CODE REVIEW

**Comment by JuanitoFatas** [comment]
@pmq20 @hlcfan @liuzxc @tzwm @happybai @diguage @alex-marmot @itsliupeng
@cassiuschen @1ternal @killernova @VincentZhao @reyesyang @jun1st @springwq
@Huibean @junzh0u @zigzagpig @42thcoder @GaaraLi @shouxian @johnnyhappy365
@aidewoode @Yunfei1982

Please review this pull request, thanks!

File: zh_cn/news/_posts/2015-04-13-ruby-openssl-hostname-matching-vulnerability.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+---
+layout: news_post
+title: "CVE-2015-1855: Ruby OpenSSL Hostname Verification"
```
Comment (hlcfan) [inline zh_cn/news/_posts/2015-04-13-ruby-openssl-hostname-matching-vulnerability.md]: Title should be in Chinese, please.

File: zh_cn/news/_posts/2015-04-13-ruby-openssl-hostname-matching-vulnerability.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+author: "zzak"
+translator: "Steven Wu"
+tags: security
+date: 2015-04-13 12:00:00 +0000
+lang: zh_cn
+---
+
+Ruby 的 OpenSSL 扩展目前存在风险，原因是其对主机名匹配的做法过于宽松，此风险会导致像 [CVE-2014-1492][CVE-2014-1492] 中提到的错误. 同样的问题也在 [Python][python-hostname-bug] 中被发现。
```
Comment (hlcfan) [inline zh_cn/news/_posts/2015-04-13-ruby-openssl-hostname-matching-vulnerability.md]: Period.

**Comment by springwq** [comment]
@hlcfan using the correct peroid now

File: zh_cn/news/_posts/2015-04-13-ruby-openssl-hostname-matching-vulnerability.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+* Subject/SAN 应该限制其只能使用 ASCII 字符
+
+所有使用受影响版本的用户应尽快升级。
+
+## 受影响的版本
+
+* 所有 Ruby 2.0 patchlevel 645 之前的版本
+* 所有 Ruby 2.1 在 2.1.6 之前的版本
+* 所有 Ruby 2.2 在 2.2.1 之前的版本
+
```
Comment (JuanitoFatas) [inline zh_cn/news/_posts/2015-04-13-ruby-openssl-hostname-matching-vulnerability.md]: There is a `prior to trunk revision 50292` missing?

https://github.com/ruby/www.ruby-lang.org/blob/master/en/news/_posts/2015-04-13-ruby-openssl-hostname-matching-vulnerability.md

File: zh_cn/news/_posts/2015-04-13-ruby-openssl-hostname-matching-vulnerability.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+## 致谢
+
+Tony Arcieri, Jeffrey Walton 和 Steffan Ullrich 报告了此问题，我们对此表示感谢！此问题首次被报告为 [Bug #9644][Bug-9644]，补丁由 Tony Arcieri 和 Horoshi Nakamura 提供。
```
Comment (JuanitoFatas) [inline zh_cn/news/_posts/2015-04-13-ruby-openssl-hostname-matching-vulnerability.md]: `Tony Arcieri, Jeffrey Walton` should use chinese 「、」。

**Comment by springwq** [comment]
@JuanitoFatas updated accordingly, please review it again.

**Comment by hlcfan** [comment]
:tada:  :confetti_ball:  Ship it!

**Comment by JuanitoFatas** [comment]
Thanks @springwq  and @hlcfan, merged manually as 34dbca2. Good work :+1:

**Comment by JuanitoFatas** [comment]
@springwq I found the title not translated, could you fix it and send another Pull Request? Thank you!

**Comment by springwq** [comment]
@JuanitoFatas  sure, will do.

**Pull request closed without merging**